stratum/templates/configmap-startup.yaml

```diff
@@ -14,7 +14,7 @@ data:
     set -e -x
     LOCAL_CHASSIS_CONFIG=$CFG_MOUNT/chassis_config.pb.txt
     if [ -f $LOCAL_CHASSIS_CONFIG ]; then
-        /usr/bin/start-stratum.sh -bf-sim -chassis_config_file=$LOCAL_CHASSIS_CONFIG {{ template "extraParams" .Values.extraParams }}
+        /usr/bin/start-stratum.sh -enable_onlp=false -chassis_config_file=$LOCAL_CHASSIS_CONFIG {{ template "extraParams" .Values.extraParams }}
```

Comment:
Any reason not to put it into `extraParams` of `values.yaml`?

Reply:
Is ONLP fully working?
If not, disabling it by default makes sense to me

Reply:
Usually it's not, disabling is fine. My comment was more about the location of the flag, not its existence.

Reply:
I consider `extraParam` more like a way to supply optional paramters / override parameters. 
If this should be disable by default across all deployments, I have no issue leaving it outside of `extraParam`